lib/mix/tasks/carbonite.gen.initial_migration.ex

```diff
@@ -64,6 +64,11 @@ defmodule Mix.Tasks.Carbonite.Gen.InitialMigration do
       #    Carbonite.Migrations.put_trigger_option("rabbits", :filtered_columns, ["private"])
       #    Carbonite.Migrations.put_trigger_option("rabbits", :mode, :ignore)
 
+      # If you wish to insert an initial outbox:
+      #
+      #    Carbonite.Migrations.create_outbox("rabbits")
+      #    Carbonite.Migrations.create_outbox("rabbits", carbonite_prefix: "carbonite_other")
+
```

Comment:
Maybe it would be nice to state that the outbox does not need to have the same name as the trigger table. 

Maybe `rabbit_holes` :P

Reply:
😄  I like it

Reply:
is it possible though that people read a sexual connotation into it?